doc/en/components/grids/_shared/row-adding.md

```diff
@@ -320,135 +320,135 @@ Then define a `{ComponentName}` with bound data source, `RowEditable` set to tru
 <!-- ComponentStart: HierarchicalGrid -->
 ```tsx
 <{ComponentSelector}
-   autoGenerate="false"
-   data={this.singersData}
+   autoGenerate={false}
```

Comment:
Could you format this snippet a bit so that it doesnt take almost half of the topic